### DISCUSSION TITLE
Store addresses in folders ?

### DISCUSSION THREAD

**Discussion by KatyPurry** [simple-login/app#889] [Q&A]
Hello,
I'm curently on free plan and I'm considering upgrading to the paid plan. 

I have 15 addresses on the home page and if I have 50 or 60, it's hard to find it. Is it possible to store these addresses in folders, for example a folder "Shopping", "hobbies", "various", etc ?

Thank you

**Comment by nguyenkims** [comment]
@Zazou49 we have indeed a feature request to better organise aliases on https://github.com/simple-login/app/discussions/398, feel free to upvote it :)
